lib/slashdeploy/deployer/fake.rb

```diff
@@ -4,18 +4,53 @@ module Deployer
     # simply records the requests in an in memory array.
     class Fake
```

Comment:
It'd probably be good to just get rid of this and add a fake Octokit::Client implementation instead. Next time.